src/soc/ibm/power9/istep_13_8.c

```diff
@@ -33,7 +33,8 @@
  */
 static void p9n_mca_scom(uint8_t chip, int mcs_i, int mca_i)
 {
-	const struct powerbus_cfg *pb_cfg = powerbus_cfg();
+	/* FIXME: don't hard-code chip number. */
+	const struct powerbus_cfg *pb_cfg = powerbus_cfg(/*chip=*/0);
```

Comment:
```suggestion
	const struct powerbus_cfg *pb_cfg = powerbus_cfg(chip);
```
Unless there is a reason to hardcode it?